x-pack/plugins/snapshot_restore/public/app/services/text/text.ts

```diff
@@ -66,22 +65,19 @@ class TextService {
     };
 
     if (type === REPOSITORY_TYPES.source && delegateType) {
-      return this.i18n.translate(
-        'xpack.snapshotRestore.repositoryType.sourceTypeWithDelegateName',
-        {
-          defaultMessage: '{delegateType} (Source-only)',
-          values: {
-            delegateType: getTypeName(delegateType),
-          },
-        }
-      );
+      return i18n.translate('xpack.snapshotRestore.repositoryType.sourceTypeWithDelegateName', {
+        defaultMessage: '{delegateType} (Source-only)',
```

Comment:
Can you check with the i18n team to see if this kind of string splitting is OK? The guidelines advise not to split sentences, but I'm not sure if this qualifies: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#splitting

Reply:
So I think this is OK, as you've really just got two phrases here.  Source-only translation would not change based on the delegateType.

Reply:
@bmcconaghy Is there any concern that the localized language's grammar may need to change the way `delegateType` is written when this noun is modified by `(Source-only)`?

Reply:
Looking at the file more closely, I actually don't think it makes sense to translate any of the `delegateType`s.  They are all proper names of technologies, and they won't really make sense translated.  Someone searching for `Azure` as translated into Japanese, for example, is going to get almost exclusively hits for things that are not Microsoft's cloud provider.  How would you translate `Hadoop HDFS`?

Reply:
I imagine the translators would just leave un-translatable types, i.e. translate `Shared file system` but leave `Azure`. if the grammar does need to change, it would be around the order of the noun and `Source-only` descriptor

Reply:
I will leave this in place for now and if there are any issues that come up in translation, we can adjust then